ts/mains/main_node.ts

```diff
@@ -790,8 +790,20 @@ async function removeDB() {
   try {
     console.error('Remove DB: removing.', userDir);
 
-    userConfig.remove();
-    ephemeralConfig.remove();
+    try {
+      userConfig.remove();
+    } catch (e) {
+      if (e.code !== 'ENOENT') {
+        throw e;
+      }
+    }
+    try {
+      ephemeralConfig.remove();
+    } catch (e) {
+      if (e.code !== 'ENOENT') {
+        throw e;
+      }
+    }
```

Comment:
Nice this has been bugging me in onboarding as well but hadn't got to debugging it!